src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working.");

```

Comment:
Similarly, the message 'Truck stopped working.' should be more informative. You might want to change it to 'Truck has stopped working.' to make it clear which machine has stopped working.